idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -161,29 +165,35 @@ class PackedColumnData(UnitModelBlockData):
         domain=float,
         description="fixed column pressure in Pa",
         doc="Fixed column operating pressure in Pa"))
+    CONFIG.declare("fix_column_pressure_drop", ConfigValue(
+        default=True,
+        domain=Bool,
+        description="Indicates whether the column pressure drop should be fixed or not",
+        doc=""""Indicates whether the column pressure drop should be fixed or not.
+If this is fixed, the column pressure profile will be calculated accordingly.
+If this is not fixed, pressure drop will be calculated based on the inbuilt correlation."""))
+    CONFIG.declare("simple_flooding_velocity_calculation", ConfigValue(
```

Comment:
I am not sure what is meant by this - from reading the doc string it makes me think that this should be a general option for providing a flooding correlation (i.e. we ask the user to explicitly provide a method, of which we have a library of some standard option). We should also allow `None` as an option for cases where flooding is not an issue.